pkg/resources/metrics.go

```diff
@@ -184,18 +184,28 @@ func createPostgresAvailabilityAlert(ctx context.Context, client k8sclient.Clien
 	productName := cr.Labels["productName"]
 	postgresCRName := strings.Title(strings.Replace(cr.Name, "postgres-example-rhmi", "", -1))
 	alertName := postgresCRName + "PostgresInstanceUnavailable"
+	sopURL := sopUrlRhoamBase + alertName + ".asciidoc"
+	alertSeverity := "critical"
+	if strings.Contains(productName, "sso") {
```

Comment:
The check is for SSO, but the comment below mentions `Marin3r`, just checking if this is correct?

Reply:
Good spot, that should be addressed now